### ISSUE TITLE
New Table: Valid Grocery Sections

### ISSUE THREAD

**Issue by verygoodsoftwarenotvirus** [dinnerdonebetter/dinnerdonebetter#279]
related to #278 

```
CREATE TABLE IF NOT EXISTS valid_grocery_sections (
    "id" TEXT NOT NULL PRIMARY KEY,
    "name" TEXT NOT NULL,
    "description" TEXT NOT NULL,
    "belongs_to_store" TEXT NOT NULL REFERENCES grocery_stores("id") ON DELETE CASCADE,
    "created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
    "last_updated_at" TIMESTAMP WITH TIME ZONE DEFAULT NULL,
    "archived_at" TIMESTAMP WITH TIME ZONE DEFAULT NULL,
    "created_by_user" TEXT NOT NULL REFERENCES users("id") ON DELETE CASCADE,
    UNIQUE("name", "belongs_to_store", "archived_at")
);
```

for things like `produce`, `poultry`, `frozen goods`, etc. We can have a generic "the store" grocery store that generic grocery sections belong to.

**Comment by verygoodsoftwarenotvirus** [comment]
blocked by #278

**Comment by verygoodsoftwarenotvirus** [comment]
closed in favor of #278
